setup.cfg

```diff
@@ -15,7 +15,6 @@ classifiers =
 packages = find:
 
 install_requires =
-    ansible
```

Comment:
I think we can make this a one line change making this `ansible-base`, I tested this earlier.

I was going to wait until `ansible-core` was out of pre-release, but we can go ahead and set to `ansible-base` for now to reduce install times.

Reply:
The problem with requiring ansible-base now is that it will make every dependent project (for example https://github.com/ansible-network/releases/blob/master/requirements.txt) have to follow the requirements of ansible-importer very closely. Eventually switching to ansible-core should be no problem (once that's properly released), but I don't think requiring ansible-base now is a good idea.

Reply:
Makes sense. Well if we don't want to just wait for `ansible-core` coming out of pre-release... this PR will need an issue in our issue tracker + changelog entry to pass CI.

Also, any code using `galaxy-importer` will need to provide it's own `ansible(|-base|-core)` until we eventually just have `ansible-core` as the requirement. So we would temporarily need to add `ansible-base` or `ansible` to `galaxy_ng` as a direct dependency... if we merge this I can handle that. The alternative is to wait on `ansible-core`

Reply:
@awcrosby now that ansible-core 2.11 has been released, it would be great if the dependency could be switched to that :)

Reply:
(Though it might make sense to wait for Ansible 4.0.0 as well, so that packages that depend both on galaxy-importer and ansible will not have problems with both ansible-base and ansible-core being installed...)

Reply:
(looking at https://github.com/ansible-network/releases/blob/master/requirements.txt :) )